src/main/java/seedu/address/model/project/Project.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.model.project;
```

Comment:
Maybe could just put project under task package as well since it is a field of a task.

Reply:
ok! should we move tag under the persons package as well?